app/src/main/java/pl/szczodrzynski/edziennik/ui/modules/settings/contributors/ContributorsActivity.kt

```diff
@@ -35,6 +35,20 @@ class ContributorsActivity : AppCompatActivity(), CoroutineScope {
     // local/private variables go here
     private val errorSnackbar: ErrorSnackbar by lazy { ErrorSnackbar(this) }
 
+    private var konami = 0
+
+    override fun onKeyUp(keyCode: Int, event: KeyEvent): Boolean {
+        return when (keyCode) {
+            KeyEvent.KEYCODE_DPAD_UP -> {konami = if (konami == 0 || konami == 1) konami.inc() else 0; true}
+            KeyEvent.KEYCODE_DPAD_DOWN -> {konami = if (konami == 2 || konami == 3) konami.inc() else 0; true}
+            KeyEvent.KEYCODE_DPAD_LEFT -> {konami = if (konami == 4 || konami == 6) konami.inc() else 0; true}
+            KeyEvent.KEYCODE_DPAD_RIGHT -> {konami = if (konami == 5 || konami == 7) konami.inc() else 0; true}
+            KeyEvent.KEYCODE_B -> {konami = if (konami == 8) konami.inc() else 0; true}
+            KeyEvent.KEYCODE_A -> {if (konami == 9) {konami += 1; b.konami.isVisible = true} else {konami = 0}; true}
+            else -> {konami = 0; super.onKeyUp(keyCode, event)}
+        }
```

Comment:
Right, it is indeed shorter, but needs to be formatted somehow. Curly braces glued to the letters just don't look nice. Also, you're using `.inc()` and then suddenly `+= 1`. You can rethink this piece a bit more clever, somehow.